### ISSUE TITLE
`tns run ios --bundle` fails on nativescript 3.4.0

### ISSUE THREAD

**Issue by erkanarslan** [NativeScript/nativescript-dev-webpack#381]
`tns run ios --bundle` command returns this error message on nativescript 3.4.0:

````
Installing...
Successfully installed on device with identifier '369FB452-C707-4F02-B757-BE2E887B71DB'.
Unable to apply changes on device: 369FB452-C707-4F02-B757-BE2E887B71DB. Error is: cp: no such file or directory: /Users/erkanarslan/Documents/Projects/cubicl-mobile/platforms/ios/cubiclmobile/app/*.
````

**Comment by NickIliev** [comment]
@erkanarslan try uninstalling the previous version of the application from your iOS device/emulator and then rebuild the application with reset `platforms` folder

**Comment by erkanarslan** [comment]
Tried it, still not working. The app folder in the error log is empty. My project's dependencies are below. If you want I can send you the whole project.

````js
"dependencies": {
    "@angular/animations": "~4.4.1",
    "@angular/common": "~4.4.1",
    "@angular/compiler": "~4.4.1",
    "@angular/core": "~4.4.1",
    "@angular/forms": "~4.4.1",
    "@angular/http": "~4.4.1",
    "@angular/platform-browser": "~4.4.1",
    "@angular/router": "~4.4.1",
    "moment": "^2.18.1",
    "nativescript-angular": "^4.4.1",
    "nativescript-checkbox": "^3.0.1",
    "nativescript-dev-version": "^0.1.2",
    "nativescript-drop-down": "^3.1.1",
    "nativescript-floatingactionbutton": "^4.1.3",
    "nativescript-loading-indicator": "^2.3.2",
    "nativescript-local-notifications": "^2.0.0",
    "nativescript-platform-css": "^1.6.5",
    "nativescript-plugin-firebase": "^5.0.5",
    "nativescript-pro-ui": "^3.2.0",
    "nativescript-ripple": "^2.0.0",
    "nativescript-theme-core": "~1.0.2",
    "nativescript-timedatepicker": "^1.2.0",
    "reflect-metadata": "~0.1.8",
    "rxjs": "^5.0.1",
    "tns-core-modules": "^3.4.0",
    "underscore": "^1.8.3",
    "zone.js": "^0.8.4"
  },
  "devDependencies": {
    "@angular/compiler-cli": "~4.4.1",
    "@ngtools/webpack": "~1.6.0",
    "babel-traverse": "6.25.0",
    "babel-types": "6.25.0",
    "babylon": "6.17.4",
    "copy-webpack-plugin": "~4.0.1",
    "extract-text-webpack-plugin": "~3.0.0",
    "lazy": "1.0.11",
    "nativescript-css-loader": "~0.26.0",
    "nativescript-dev-typescript": "~0.5.0",
    "nativescript-dev-webpack": "^0.8.0",
    "nativescript-worker-loader": "~0.8.1",
    "raw-loader": "~0.5.1",
    "resolve-url-loader": "~2.1.0",
    "tslib": "^1.7.1",
    "typescript": "^2.6.2",
    "webpack": "~3.2.0",
    "webpack-bundle-analyzer": "^2.8.2",
    "webpack-sources": "~1.0.1"
  }
````

**Comment by berchik** [comment]
`tns run ios --bundle` executes ok for me but `tns run ios --bundle --env.aot` doesn't (on android too). I use angular 5.1.0, tns-ios 3.4.0, tns-core-modules 3.4.0, tns-core-modules 0.9.0.
Also, @NickIliev is correct about the need to uninstall the previous version from the simulator.

**Comment by NickIliev** [comment]
@erkanarslan you will need to update your Angular and Webpack dependencies - full guide on how to migrate to 3.4.0 can be found [here](https://www.nativescript.org/blog/nativescript-webpack-0.9.0-what-changed-and-how-to-upgrade)

@berchik there is a known bug with AOT builds on Windows logged [here](https://github.com/NativeScript/nativescript-dev-webpack/issues/376) - if you are using Windows pelase stay tuned for updates and upcoming fix. If that is not the case please send me more details about how to reproduce the issue (preferably in a new separate issue)

**Comment by berchik** [comment]
@NickIliev windows only. thanks.

**Comment by erkanarslan** [comment]
After upgrading the dependencies, bundling works. Thanks.

**Comment by Gitelaus** [comment]
I'm having this issue after creating the project via the latest `nativescript-schematics` + `ng new -c=@nativescript/schematics`.

My versions as follows:

```
// dependencies...
"nativescript-angular": "~6.1.0",
// dev-dependencies
"nativescript-dev-sass": "^1.6.0",
"nativescript-dev-typescript": "~0.7.0",
"nativescript-dev-webpack": "^0.15.1",
```

and my global installation of `nativescript` is `nativescript@4.2.3`.

The error I am getting is the same as the original issue posters.

Am I running an out of date dependencies, or am I missing a reference somewhere?

**Comment by NickIliev** [comment]
@Gitelaus try updating your `nativescript-dev-webpack` and the related config file. Upgrade instructions [here](https://docs.nativescript.org/angular/releases/upgrade-instructions#updating-webpack-version-and-configuration)

**Comment by Gitelaus** [comment]
@NickIliev thanks for the response. Funnily enough, I fixed this by downgrading from `0.15.1` of `nativescript-dev-webpack` to `0.15.0`. Unsure exactly what was going on, but it looks like it wasn't doing the actual build of files.
